atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -340,6 +340,9 @@ def run(self):
             self.log.info("fetching image %s from docker", image)
             with tempfile.NamedTemporaryFile(prefix='docker-image-', suffix='.tar') as image_file:
                 image_file.write(self.tasker.d.get_image(image).data)
+                # This file will be referenced by its filename, not file
+                # discriptor - must ensure contents are written to disk
```

Comment:
Typo, 'descriptor'.
Has this bug been lurking all along? Would it be better in its own commit (and may PR)?